packages/@aws-cdk/aws-batch/lib/compute-environment.ts

```diff
@@ -0,0 +1,484 @@
+   *
+   * The maximum percentage that a Spot Instance price can be when compared with the On-Demand price for
+   * that instance type before instances are launched. For example, if your maximum percentage is 20%,
+   * then the Spot price must be below 20% of the current On-Demand price for that EC2 instance. You always
+   * pay the lowest (market) price and never more than your maximum percentage. If you leave this field empty,
+   * the default value is 100% of the On-Demand price.
+   *
+   * @default 100
+   */
+  readonly bidPercentage?: number;
+
+  /**
+   * The desired number of EC2 vCPUS in the compute environment.
+   *
+   * @default - no desired vcpu value will be used
+   */
+  readonly desiredvCpus?: number;
+
+  /**
+   * The maximum number of EC2 vCPUs that an environment can reach. Each vCPU is equivalent to
+   * 1,024 CPU shares. You must specify at least one vCPU.
+   *
+   * @default 256
+   */
+  readonly maxvCpus?: number;
+
+  /**
+   * The minimum number of EC2 vCPUs that an environment should maintain (even if the compute environment state is DISABLED).
+   * Each vCPU is equivalent to 1,024 CPU shares. You must specify at least one vCPU.
+   *
+   * @default 1
+   */
+  readonly minvCpus?: number;
+
+  /**
+   * The EC2 key pair that is used for instances launched in the compute environment.
+   * If no key is defined, then SSH access is not allowed to provisioned compute resources.
+   *
+   * @default - No SSH access will be possible.
+   */
+  readonly ec2KeyPair?: string;
+
+  /**
+   * The Amazon Machine Image (AMI) ID used for instances launched in the compute environment.
+   *
+   * @default - no image will be used
+   */
+  readonly image?: ec2.IMachineImage;
+
+  /**
+   * The launch template to use for your compute resources. Any other compute resource parameters
+   * that you specify in a CreateComputeEnvironment API operation override the same parameters in
+   * the launch template. You must specify either the launch template ID or launch template name in
+   * the request, but not both. For more information, see Launch Template Support in the AWS Batch User Guide.
+   *
+   * @default - no launch template will be used
+   */
+  readonly launchTemplate?: ec2.CfnInstance.LaunchTemplateSpecificationProperty;
+
+  /**
+   * The Amazon EC2 placement group to associate with your compute resources. If you intend to submit multi-node
+   * parallel jobs to your compute environment, you should consider creating a cluster placement group and
+   * associate it with your compute resources. This keeps your multi-node parallel job on a logical grouping of
+   * instances within a single Availability Zone with high network flow potential. For more information, see
+   * Placement Groups in the Amazon EC2 User Guide for Linux Instances.
+   *
+   * @default - no placement group will be used
+   */
+  readonly placementGroup?: ec2.CfnPlacementGroup;
```

Comment:
Like I said - please remove the `launchTemplate` and `placementGroup` properties for now.

Reply:
Got it 👍